assets/pages/subscriptions-landing/subscriptionsLandingReducer.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { combineReducers } from 'redux';
+
+import { contributionSelectionReducerFor } from 'containerisableComponents/contributionSelection/contributionSelectionReducer';
+import { payPalContributionButtonReducerFor } from 'containerisableComponents/payPalContributionButton/payPalContributionButtonReducer';
+
+import type { State as ContributionSelectionState } from 'containerisableComponents/contributionSelection/contributionSelectionReducer';
+import type { State as PayPalState } from 'containerisableComponents/payPalContributionButton/payPalContributionButtonReducer';
+
+import type { CommonState } from 'helpers/page/page';
+
+
+// ----- Types ----- //
+
+type PageState = {
+  selection: ContributionSelectionState,
+  payPal: PayPalState,
+};
+
+export type State = {
+  common: CommonState,
+  page: PageState,
+};
+
+
+// ----- Reducer ----- //
+
+export default combineReducers({
+  selection: contributionSelectionReducerFor('CONTRIBUTE_SECTION', 'GBPCountries'),
+  payPal: payPalContributionButtonReducerFor('CONTRIBUTE_SECTION'),
+});
```

Comment:
I think you can probably do away with most of this file. There's no contributions section on the page any more, so there's no state outside the common state. We haven't actually had this situation before, so there's no convention for how we handle this.

Currently the [`page.init`](https://github.com/guardian/support-frontend/blob/master/assets/helpers/page/page.js#L123) function expects *some* page state and so creates a nested object with both `page` and `common` fields. I think you probably have two options for handling this:

1. Create the `State` type in `subscriptionsLanding.jsx` and pass an empty object to the `page.init` function. This will create a `page` field in the Redux state that consists of just an empty object. Quicker but not ideal.
2. Refactor `page.init` to handle cases where there's no `pageReducer`, because we still want the common state and AB test/analytics initialisation.

Shall we have a chat about this offline?